universe/UniverseObject.h

```diff
@@ -29,6 +29,10 @@ FO_COMMON_API extern const int INVALID_OBJECT_ID;
 // The ID number assigned to temporary universe objects
 FO_COMMON_API extern const int TEMPORARY_OBJECT_ID;
 
+// The ID number assigned instead of the system which represents the whole imperium
+// Used for industrial stockpile
+FO_COMMON_API extern const int IMPERIAL_OBJECT_ID;
```

Comment:
@agrrr3, bear in mind that *empires* are not *objects* like planets, ships, etc. are. The latter are all subclasses of `UniverseObject`, and AFAIK all universe objects share the same "id pool" (so there can't be a ship and a planet with the same id).

However, as empires aren't universe objects, they have an independent "id pool", so, as @geoffthemedio already said, there can be an empire with the same id as a universe object (ship, planet, etc.).